init.sh

```diff
@@ -1,5 +1,28 @@
 #!/usr/bin/env sh
 
+function install_docker () {
+    echo "Installing Docker. Press ctrl-c to quit"
+    # Give the user time to respond
+    sleep 5
+    curl https://get.docker.com/ | sh
+}
+
+which docker
+
+if [ $? -eq 0 ]
+then
+    docker --version | grep "Docker version"
+    if [ $? -eq 0 ]
+    then
+        echo "Docker is installed. *Proceed*"
```

Comment:
This will get printed every time ./tc is called, so the output wouldn't be relevant in most cases. Could you instead negate your if statement, notify the user that Docker isn't installed, and call install_docker?

Reply:
Absolutely. Thanks  for mentioning this. I made some adjustments, to reduce expensive echo statements.